front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,272 @@
+      { src: IBMPlexMonoMedium, fontWeight: 'medium' },
+      { src: IBMPlexMonoBold, fontWeight: 'bold' },
+    ],
+  });
+
+  return (
+    <Document>
+      <Page wrap={false} style={styles.page} size={[1344]}>
+        <View style={styles.alertBanner}>
+          <Image src={iconAlert} style={styles.alertIcon} />
+          <Text style={styles.simulationTitle}>{t('simulation')}</Text>
+          <Text style={styles.message}>{t('warningMessage')}</Text>
+        </View>
+        <View>
+          <View style={styles.numberDateBanner}>
+            <View style={styles.stdcm}>
+              <Text style={styles.title}>{t('stdcm')}</Text>
+              <Text style={styles.creation}>{t('stdcmCreation')}</Text>
+            </View>
+            <View style={styles.numericInfo}>
+              <Text style={styles.number}>
+                {t('number')}
+                {number}
+              </Text>
+              <Text style={styles.creationDate}>{formatDate(stdcmData?.path.created)}</Text>
+            </View>
+            <Image src={logoSNCF} style={styles.sncfLogo} />
+          </View>
+        </View>
+        <View style={styles.rcInfo}>
+          <View style={styles.rcBox}>
+            <Text style={styles.rcName}>Super Fret</Text>
+            <Text style={styles.rcPersonName}>Stéphanie Martin-Duclotte</Text>
+          </View>
+          <View style={styles.rcBox}>
+            <Text style={styles.rcPhoneNumber}>04 12 34 56 78</Text>
+            <Text style={styles.rcMail}>stephanie.martin-duclotte@superfret.fr</Text>
+          </View>
+          <View style={styles.rcBox}>
+            <View style={styles.stdcmApplication}>
+              <Text style={styles.applicationDate}>{t('applicationDate')}</Text>
+              <Text style={styles.date}>{formatDay()}</Text>
+              <Text style={styles.referencePath}>{t('referencePath')}</Text>
+              <Text style={styles.pathNumber}>n°123456</Text>
+            </View>
+          </View>
+        </View>
+        <View style={styles.convoyAndRoute}>
+          <View style={styles.convoy}>
+            <Text style={styles.convoyTitle}> {t('convoy')}</Text>
+            <View style={styles.convoyInfo}>
+              <View style={styles.convoyInfoBox1}>
+                <Text style={styles.convoyInfoTitles}>{t('speedLimitByTag')}</Text>
+                <Text style={styles.convoyInfoData}>
+                  {stdcmData?.simulation.speed_limit_tags === null
+                    ? '-'
+                    : stdcmData?.simulation.speed_limit_tags}
+                </Text>
+                <Text style={styles.convoyInfoTitles}>{t('towedMaterial')}</Text>
+                <Text style={styles.convoyInfoData}>-</Text>
+                <Text style={styles.convoyInfoTitles}>{t('maxSpeed')}</Text>
+                <Text style={styles.convoyInfoData}>
+                  {rollingStockData?.max_speed && rollingStockData?.max_speed * 3.6} {t('kmH')}
```

Comment:
I think it would be better to use a string that you complete and no need to translate the units :)

Same for:
- mass
- length

```suggestion
                  {`${rollingStock.max_speed * 3.6} km/h`}
```

Reply:
done